app/src/main/java/com/example/synerzip/recircle_android/models/User_addresses.java

```diff
@@ -0,0 +1,26 @@
+package com.example.synerzip.recircle_android.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by Snehal Tembare on 9/3/17.
+ * Copyright © 2017 Synerzip. All rights reserved
+ */
+
+@Getter
+@Setter
+
+public class User_addresses {
```

Comment:
Start a new word with uppercase in class name, and there should be no special  character between 2 words